src/wallet/rpcwallet.cpp

```diff
@@ -3785,8 +3788,9 @@ RPCHelpMan getaddressinfo()
                         {RPCResult::Type::STR_HEX, "pubkey", /* optional */ true, "The hex value of the raw public key for single-key addresses (possibly embedded in P2SH or P2WSH)."},
                         {RPCResult::Type::OBJ, "embedded", /* optional */ true, "Information about the address embedded in P2SH or P2WSH, if relevant and known.",
                         {
-                            {RPCResult::Type::ELISION, "", "Includes all getaddressinfo output fields for the embedded address, excluding metadata (timestamp, hdkeypath, hdseedid)\n"
-                            "and relation to the wallet (ismine, iswatchonly)."},
+                            {RPCResult::Type::ELISION, "",
+                                    "Includes all getaddressinfo output fields for the embedded address, excluding metadata (timestamp,\n"
+                                    "hdkeypath, hdseedid) and relation to the wallet (ismine, iswatchonly)."},
```

Comment:
Fixup requested in https://github.com/bitcoin/bitcoin/pull/19200#discussion_r438781980.